source/_integrations/shelly.markdown

```diff
@@ -248,6 +248,32 @@ shelly:
   coap_port: 12345
 ```
 
+## Network and Firewall
```

Comment:
I totally agree. At best it could be a general "networking" page, but honestly I would not go that way.

Reply:
Although this is a very important information I tend to go with @frenck & @chemelli74, adding this to the docs will make most users confused.
Since you already did the work documenting it, I would highly applicate if you post it on the community forums as this will be the place users will go to search if they need this information. https://community.home-assistant.io/
Thanks.